besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -401,12 +401,18 @@ private void detectJemalloc(final List<String> lines) {
         .ifPresentOrElse(
             t -> {
               try {
-                final String version = PlatformDetector.getJemalloc();
-                lines.add("jemalloc: " + version);
+                String jemallocEnabled = environment.get("BESU_USING_JEMALLOC");
+                if (jemallocEnabled.equals("true")) {
+                  final String version = PlatformDetector.getJemalloc();
+                  lines.add("jemalloc: " + version);
+                } else {
+                  logger.info(
+                      "BESU_USING_JEMALLOC is present but is not set to true, jemalloc library not loaded");
+                }
+
               } catch (final Throwable throwable) {
                 logger.warn(
-                    "BESU_USING_JEMALLOC is present but we failed to load jemalloc library to get the version",
-                    throwable);
+                    "BESU_USING_JEMALLOC is present but we failed to load jemalloc library to get the version");
```

Comment:
why remove the throwable from this log?

Reply:
> why remove the throwable from this log?

Well, I tested all scenarios and in this particular one, since it is logged as a WARN, I thought the logs got too clogged for a simple warning...  Instead of only have 

```
2024-05-16 13:22:14.663+01:00 | main | WARN  | Besu | BESU_USING_JEMALLOC is present but we failed to load jemalloc library to get the version
```

I get 😮

```
2024-05-16 13:22:14.663+01:00 | main | WARN  | Besu | BESU_USING_JEMALLOC is present but we failed to load jemalloc library to get the version
java.lang.UnsatisfiedLinkError: Unable to load library 'jemalloc':
libjemalloc.so: cannot open shared object file: No such file or directory
libjemalloc.so: cannot open shared object file: No such file or directory
Native library (linux-x86-64/libjemalloc.so) not found in resource path (/home/amsmota/Development/besu/build/install/besu/lib/besu-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-evmtool-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-ethstats-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-clique-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ibft-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-qbft-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-consensus-common-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-retesteth-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-stratum-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-api-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-merge-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-blockcreation-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-eth-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-permissioning-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-p2p-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-plugin-rocksdb-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/referencetests-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-core-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/core-support-test-24.5-develop-5affb81-test-support.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-config-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-trie-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-crypto-services-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-pipeline-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-tasks-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-metrics-rocksdb-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-metrics-core-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-kvstore-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/plugin-api-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-evm-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-datatypes-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-pki-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/enclave-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-nat-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-util-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/oshi-core-6.4.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/graphql-java-21.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-crypto-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/core-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/dnsjava-3.5.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-dns-discovery-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-devp2p-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-slf4j2-impl-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/java-dataloader-3.2.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/crypto-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/Java-WebSocket-1.5.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/jose4j-0.9.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/slf4j-api-2.0.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-rlp-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-annotations-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-web-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-auth-jwt-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-unit-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-web-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-auth-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-bridge-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-lang-kotlin-coroutines-4.3.7.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-lang-kotlin-4.3.7.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-core-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-codegen-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-datatype-jdk8-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-core-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-databind-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_guava-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-net-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-merkle-trie-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-crypto-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-kv-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-io-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/discovery-22.12.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-netty-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-core-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-kademlia-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-context-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-api-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-util-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/guava-33.0.0-jre.jar:/home/amsmota/Development/besu/build/install/besu/lib/dagger-2.50.jar:/home/amsmota/Development/besu/build/install/besu/lib/picocli-4.7.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/maven-artifact-3.9.6.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-api-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-compress-1.26.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-lang3-3.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-core-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-units-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-rlp-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-bytes-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-config-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-toml-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/spring-security-crypto-6.2.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/snappy-java-1.1.10.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/jc-kzg-4844-1.0.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/rocksdbjni-8.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-jul-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/splunk-library-javalogging-1.11.8.jar:/home/amsmota/Development/besu/build/install/besu/lib/jansi-2.4.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/value-annotations-2.10.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcpkix-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/abi-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-api-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-extension-trace-propagators-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-extension-autoconfigure-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-otlp-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-extension-autoconfigure-spi-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-trace-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-metrics-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-logs-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-sender-okhttp-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-otlp-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-extension-incubator-1.33.0-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-api-events-1.33.0-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-api-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr4-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr4-runtime-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcutil-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/rlp-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/utils-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcprov-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/caffeine-3.1.8.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-all-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-riscv64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final-osx-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final-osx-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/bls12-381-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/arithmetic-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jna-platform-5.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/secp256k1-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/secp256r1-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/blake2bf-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jna-5.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-concurrent-coroutines-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-concurrent-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-proxy-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-http2-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-http-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-native-macos-4.1.104.Final-osx-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-native-macos-4.1.104.Final-osx-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-classes-macos-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-classes-epoll-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-classes-kqueue-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-unix-common-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_pushgateway-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_httpserver-0.15.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_common-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_hotspot-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlinx-coroutines-jdk8-1.6.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlinx-coroutines-core-jvm-1.6.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/logging-interceptor-4.10.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/okhttp-4.12.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/okio-jvm-3.6.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-jdk8-1.9.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-jdk7-1.9.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-1.9.22.jar:/home/amsmota/Development/besu/build/install/besu/lib/encoder-1.2.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-tcnative-boringssl-static-2.0.62.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-semconv-1.23.1-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.jupnp-2.7.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.jupnp.support-2.7.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/failureaccess-1.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/amsmota/Development/besu/build/install/besu/lib/jsr305-3.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/checker-qual-3.41.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/error_prone_annotations-2.23.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.inject-1.jar:/home/amsmota/Development/besu/build/install/besu/lib/rxjava-2.2.21.jar:/home/amsmota/Development/besu/build/install/besu/lib/reactor-core-3.4.24.jar:/home/amsmota/Development/besu/build/install/besu/lib/reactive-streams-1.0.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-socks-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-dns-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-buffer-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-common-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/framework-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/framework-internal-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/gson-fire-1.8.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/gson-2.10.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-net-3.9.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-context-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuples-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-unixsocket-0.38.17.jar:/home/amsmota/Development/besu/build/install/besu/lib/ens-normalize-0.1.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/plexus-utils-3.5.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_otel-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_otel_agent-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-enxio-0.32.13.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-posix-3.1.15.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-ffi-2.2.13.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-codec-1.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/annotations-13.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/perfmark-api-0.26.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/annotations-4.1.1.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/animal-sniffer-annotations-1.23.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-tcnative-classes-2.0.62.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-haproxy-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-memcache-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-mqtt-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-redis-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-smtp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-stomp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-xml-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-ssl-ocsp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-rxtx-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-sctp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-udt-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-proto-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/snakeyaml-2.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-io-2.15.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/protobuf-java-3.22.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-collections4-4.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/ST4-4.3.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr-runtime-3.5.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.abego.treelayout.core-1.0.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.json-1.1.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/icu4j-71.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-constants-0.10.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_common-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jffi-1.3.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/jffi-1.3.10-native.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-commons-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-util-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-analysis-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-tree-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-a64asm-1.0.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-x86asm-1.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.persistence-api-2.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.annotation-api-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/swagger-annotations-1.6.9.jar)
        at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:325)
        at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:481)
        at com.sun.jna.Library$Handler.<init>(Library.java:197)
        at com.sun.jna.Native.load(Native.java:618)
        at com.sun.jna.Native.load(Native.java:592)
        at org.hyperledger.besu.util.platform.PlatformDetector.detectJemalloc(PlatformDetector.java:340)
        at org.hyperledger.besu.util.platform.PlatformDetector.getJemalloc(PlatformDetector.java:115)
        at org.hyperledger.besu.cli.ConfigurationOverviewBuilder.lambda$detectJemalloc$0(ConfigurationOverviewBuilder.java:406)
        at java.base/java.util.Optional.ifPresentOrElse(Optional.java:196)
        at org.hyperledger.besu.cli.ConfigurationOverviewBuilder.detectJemalloc(ConfigurationOverviewBuilder.java:401)
        at org.hyperledger.besu.cli.ConfigurationOverviewBuilder.build(ConfigurationOverviewBuilder.java:382)
        at org.hyperledger.besu.cli.BesuCommand.generateConfigurationOverview(BesuCommand.java:2790)
        at org.hyperledger.besu.cli.BesuCommand.configure(BesuCommand.java:1780)
        at org.hyperledger.besu.cli.BesuCommand.run(BesuCommand.java:1151)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at org.hyperledger.besu.cli.BesuCommand.lambda$createExecuteTask$1(BesuCommand.java:1078)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at org.hyperledger.besu.cli.BesuCommand.lambda$createPluginRegistrationTask$2(BesuCommand.java:1088)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at org.hyperledger.besu.cli.util.ConfigDefaultValueProviderStrategy.execute(ConfigDefaultValueProviderStrategy.java:58)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at org.hyperledger.besu.cli.BesuCommand.executeCommandLine(BesuCommand.java:1115)
        at org.hyperledger.besu.cli.BesuCommand.parse(BesuCommand.java:1057)
        at org.hyperledger.besu.Besu.main(Besu.java:39)
        Suppressed: java.lang.UnsatisfiedLinkError: libjemalloc.so: cannot open shared object file: No such file or directory
                at com.sun.jna.Native.open(Native Method)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:213)
                ... 30 more
        Suppressed: java.lang.UnsatisfiedLinkError: libjemalloc.so: cannot open shared object file: No such file or directory
                at com.sun.jna.Native.open(Native Method)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:226)
                ... 30 more
        Suppressed: java.io.IOException: Native library (linux-x86-64/libjemalloc.so) not found in resource path (/home/amsmota/Development/besu/build/install/besu/lib/besu-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-evmtool-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-ethstats-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-clique-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ibft-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-qbft-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-consensus-common-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-retesteth-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-stratum-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-api-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-merge-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-blockcreation-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-eth-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-permissioning-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-p2p-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-plugin-rocksdb-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/referencetests-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-core-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/core-support-test-24.5-develop-5affb81-test-support.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-config-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-trie-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-crypto-services-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-pipeline-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-tasks-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-metrics-rocksdb-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-metrics-core-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-kvstore-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/plugin-api-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-evm-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-datatypes-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-pki-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/enclave-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-nat-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-util-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/oshi-core-6.4.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/graphql-java-21.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-crypto-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/core-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/dnsjava-3.5.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-dns-discovery-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-devp2p-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-slf4j2-impl-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/java-dataloader-3.2.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/crypto-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/Java-WebSocket-1.5.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/jose4j-0.9.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/slf4j-api-2.0.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/besu-ethereum-rlp-24.5-develop-5affb81.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-annotations-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-web-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-auth-jwt-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-unit-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-web-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-auth-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-bridge-common-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-lang-kotlin-coroutines-4.3.7.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-lang-kotlin-4.3.7.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-core-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/vertx-codegen-4.5.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-datatype-jdk8-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-core-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jackson-databind-2.16.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_guava-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-net-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-merkle-trie-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-crypto-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-kv-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-io-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/discovery-22.12.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-netty-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-core-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-kademlia-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-context-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-api-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/grpc-util-1.60.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/guava-33.0.0-jre.jar:/home/amsmota/Development/besu/build/install/besu/lib/dagger-2.50.jar:/home/amsmota/Development/besu/build/install/besu/lib/picocli-4.7.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/maven-artifact-3.9.6.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-api-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-compress-1.26.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-lang3-3.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-core-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-units-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-rlp-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-bytes-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-config-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-toml-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/spring-security-crypto-6.2.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/snappy-java-1.1.10.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/jc-kzg-4844-1.0.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/rocksdbjni-8.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-jul-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/splunk-library-javalogging-1.11.8.jar:/home/amsmota/Development/besu/build/install/besu/lib/jansi-2.4.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/value-annotations-2.10.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcpkix-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/abi-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/log4j-api-2.22.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-extension-trace-propagators-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-extension-autoconfigure-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-otlp-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-extension-autoconfigure-spi-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-trace-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-metrics-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-logs-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-sender-okhttp-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-sdk-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-otlp-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-extension-incubator-1.33.0-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-api-events-1.33.0-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-exporter-common-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-api-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr4-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr4-runtime-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcutil-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/rlp-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/utils-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/bcprov-jdk18on-1.77.jar:/home/amsmota/Development/besu/build/install/besu/lib/caffeine-3.1.8.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-all-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-epoll-4.1.104.Final-linux-riscv64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final-osx-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-kqueue-4.1.104.Final-osx-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/bls12-381-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/arithmetic-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jna-platform-5.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/secp256k1-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/secp256r1-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/blake2bf-0.8.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jna-5.14.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-concurrent-coroutines-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuweni-concurrent-2.4.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-proxy-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-http2-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-http-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-native-macos-4.1.104.Final-osx-x86_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-native-macos-4.1.104.Final-osx-aarch_64.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-classes-macos-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-dns-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-classes-epoll-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-classes-kqueue-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-native-unix-common-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_pushgateway-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_httpserver-0.15.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_common-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_hotspot-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlinx-coroutines-jdk8-1.6.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlinx-coroutines-core-jvm-1.6.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/logging-interceptor-4.10.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/okhttp-4.12.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/okio-jvm-3.6.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-jdk8-1.9.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-jdk7-1.9.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/kotlin-stdlib-1.9.22.jar:/home/amsmota/Development/besu/build/install/besu/lib/encoder-1.2.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-tcnative-boringssl-static-2.0.62.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-semconv-1.23.1-alpha.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.jupnp-2.7.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.jupnp.support-2.7.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/failureaccess-1.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/home/amsmota/Development/besu/build/install/besu/lib/jsr305-3.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/checker-qual-3.41.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/error_prone_annotations-2.23.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.inject-1.jar:/home/amsmota/Development/besu/build/install/besu/lib/rxjava-2.2.21.jar:/home/amsmota/Development/besu/build/install/besu/lib/reactor-core-3.4.24.jar:/home/amsmota/Development/besu/build/install/besu/lib/reactive-streams-1.0.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-socks-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-dns-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-buffer-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-resolver-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-common-4.1.107.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/framework-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/framework-internal-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/gson-fire-1.8.5.jar:/home/amsmota/Development/besu/build/install/besu/lib/gson-2.10.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-net-3.9.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/opentelemetry-context-1.33.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/tuples-4.11.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-unixsocket-0.38.17.jar:/home/amsmota/Development/besu/build/install/besu/lib/ens-normalize-0.1.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/plexus-utils-3.5.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_otel-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_otel_agent-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-enxio-0.32.13.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-posix-3.1.15.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-ffi-2.2.13.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-codec-1.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/annotations-13.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/perfmark-api-0.26.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/annotations-4.1.1.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/animal-sniffer-annotations-1.23.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-tcnative-classes-2.0.62.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-haproxy-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-memcache-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-mqtt-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-redis-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-smtp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-stomp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-codec-xml-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-handler-ssl-ocsp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-rxtx-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-sctp-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/netty-transport-udt-4.1.104.Final.jar:/home/amsmota/Development/besu/build/install/besu/lib/client-java-proto-18.0.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/snakeyaml-2.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-io-2.15.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/protobuf-java-3.22.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/commons-collections4-4.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/ST4-4.3.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/antlr-runtime-3.5.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/org.abego.treelayout.core-1.0.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.json-1.1.4.jar:/home/amsmota/Development/besu/build/install/besu/lib/icu4j-71.1.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-constants-0.10.3.jar:/home/amsmota/Development/besu/build/install/besu/lib/simpleclient_tracer_common-0.16.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jffi-1.3.10.jar:/home/amsmota/Development/besu/build/install/besu/lib/jffi-1.3.10-native.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-commons-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-util-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-analysis-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-tree-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/asm-9.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-a64asm-1.0.0.jar:/home/amsmota/Development/besu/build/install/besu/lib/jnr-x86asm-1.0.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.persistence-api-2.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/javax.annotation-api-1.3.2.jar:/home/amsmota/Development/besu/build/install/besu/lib/swagger-annotations-1.6.9.jar)
                at com.sun.jna.Native.extractFromResourcePath(Native.java:1141)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:297)
                ... 30 more
```

Reply:
makes sense